Shopbeam order #<%= @order.id %> has been completed with the error:

<%= @exception.message %>
